gokart/task.py

```diff
@@ -279,6 +280,36 @@ def get_task_params(self) -> Dict:
             return self.load(target)
         return dict()
 
+    @luigi.Task.event_handler(luigi.Event.START)
+    def _set_random_seed(self):
+        random_seed = self._get_random_seed()
+        seed_methods = self.try_set_seed(self.fix_random_seed_methods, random_seed)
+        self.dump({'seed': random_seed, 'seed_methods': seed_methods}, self._get_random_seeds_target())
```

Comment:
Cool way to reproduce results.

IMO, there must be another way to inject random seeds because  task id occasionally changes.
How about introducing another params like `fix_random_seed_value`?

Reply:
That's right. It looks like a `fixed task_id`. I'll try :)

Reply:
done https://github.com/m3dev/gokart/pull/128/commits/068c50819ffa0b401e1be8451b84d3f71715fc64

Reply:
@Hi-king please check PR!

Reply:
@vaaaaanquish cool way!